src/accountsdb/db.zig

```diff
@@ -401,7 +403,7 @@ pub const AccountsDB = struct {
             var thread_db = loading_threads.addOneAssumeCapacity();
             thread_db.* = try AccountsDB.init(
                 per_thread_allocator,
-                .noop, // dont spam the logs with init information (we set it after)
+                Logger{ .noop = {} }, // dont spam the logs with init information (we set it after)
```

Comment:
Slight preference due to it being clear it's a logger being passed `AccountsDB.init(.., .noop, ...)` may not be clear what `.noop` represents. But I also do see the appeal for `.noop` and since this is the style already, i'll revert to that.

Reply:
That's reasonable. You can always specify an enum type for clarity, it would look like `Logger.noop`.